src/components/01-branding/02-typography/fields/fields-default.njk

```diff
@@ -0,0 +1,67 @@
+<table class="usa-table">
+  <caption>Fields</caption>
+  <thead>
+    <tr>
+      <th scope="col">Style</th>
+      <th scope="col">Markup</th>
+      <th scope="col">Configuration</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <td><element class="sds-field"><h1>Field Header</h1></element></td>
+      <th scope="row">&ltelement class="sds-field">&lth1>&lt/h1>&lt/element></th>
+      <td>Weight: Semi-Bold / Size: 1.46rem / Line Height: 1.1rem / Color: Base-Dark (#454540)</td>
+    </tr>
+    <tr>
+      <td><element class="sds-field"><h1 class="sds-light">Field Header</h1></element></td>
```

Comment:
All of the h# elements here extend h2, so the sizes will be the same
Christy wanted to use sds-light to keep the styles the same, since it's what we're using for the headings